app/code/Mykhailok/AskAboutThisProduct/view/frontend/web/template/ask-question-form.html

```diff
@@ -0,0 +1,57 @@
+<div id="mykhailok-question-form">
```

Comment:
@mykhailok-dv 
1) Critical issue! Form can be submitted without any validation
![Selection_304](https://user-images.githubusercontent.com/13600382/117792572-125cca00-b254-11eb-99b2-5c1aea23b5d3.png)

2) The form must be added as a new tab to the product page. It should not be located under the tabs.